Update step 1 (#12)
aws-samples/aws-waf-workshop: docs/step-1.md

# Step 1 - Getting Started with AWS WAF Security Automations Solution

## Introduction

> In this Step we will add the Application Load Balancer (ALB) of the sample application with the AWS WAF. Next, we will explore how the AWS WAF can protect against malicious requests containing SQL Injection or Cross Site Scripting attacks. Finally, we demonstrate how the Security Automation Solution updates the AWS WAF ruleset to protect against HTTP Floods, Scanners and Probes.

Reviewer: ```suggestion
In this Step you will associate AWS WAF with the Application Load Balancer (ALB) of the sample application, then explore how the AWS WAF can protect against malicious requests containing SQL Injection or Cross Site Scripting attacks, and finally examine how the Security Automation Solution updates the AWS WAF ruleset to protect against HTTP Floods, Scanners and Probes.
```

## 1.1 Associate the solution's AWS WAF Web ACL with your ALB

> The AWS WAF can be used with either API Gateway, Amazon CloudFront or an Application Load Balancer. In this example, we are using AWS WAF with an Application Load Balancer. Before the AWS WAF can recieve requests, we must add the Application Load Balancer to the AWS WAF Resource.

Reviewer: ```suggestion
The AWS WAF can be used with either API Gateway, Amazon CloudFront or an Application Load Balancer. In this example, we are using AWS WAF with an Application Load Balancer. Before the AWS WAF can receive requests, you must add the Application Load Balancer to the AWS WAF Resource.
```

"i before e except after c" 🙂

## 1.3 HTTP Flood (AWS Lambda log parser)

> HTTP Flood is a style of attack where an application recieves a large volume of legitimate requests, intended to overwhelm the application.

Reviewer: ```suggestion
HTTP floods are a type of denial-of-service attack where an application receives a large volume of requests intended to exhaust the resources of an application, preventing it from handling requests from legitimate users.
```

> We will demonstrate functionality provided by the Security Automation Solution to detect these style of attacks and block the responsible IP address in AWS WAF.

Reviewer: ```suggestion
We will demonstrate functionality provided by the Security Automation Solution to detect this type of attack and block the responsible IP address in AWS WAF.
```
